src/Illuminate/Database/Eloquent/BroadcastsEvents.php

```diff
@@ -26,6 +26,10 @@ public static function bootBroadcastsEvents()
                 $model->broadcastTrashed();
             });
 
+            static::willTrash(function ($model) {
+                $model->broadcastWillTrash();
+            });
+
```

Comment:
Added broadcasting for the `willTrash` event, to keep consistency with the `trashed` event.